LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
Please don't remove the license.

Reply:
Will bring it back, I think this was accidental. The plan for the wheels however was to link to the fpm README (for long description) and the LICENSE, so that we don't maintain an explicit copy of them in this repo.